src/Models/Link.php

```diff
@@ -40,30 +41,60 @@ class Link extends DataObject implements JsonData, Type
      */
     private ?string $linkType = null;
 
+    /**
+     * @deprecated 3.0.0 Will be removed without equivalent functionality to replace it
+     */
     public function defineLinkTypeRequirements()
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('3.0.0', 'Will be removed without equivalent functionality to replace it.');
+        });
         Requirements::add_i18n_javascript('silverstripe/linkfield:client/lang', false, true);
         Requirements::javascript('silverstripe/linkfield:client/dist/js/bundle.js');
         Requirements::css('silverstripe/linkfield:client/dist/styles/bundle.css');
     }
 
+    /**
+     * @deprecated 3.0.0 Will be renamed to getLinkTypeHandlerName() is linkfield v4
+     */
     public function LinkTypeHandlerName(): string
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('3.0.0', 'Will be renamed to getLinkTypeHandlerName() is linkfield v4.');
+        });
         return 'FormBuilderModal';
     }
 
+    /**
+     * @deprecated 3.0.0 Will be removed without equivalent functionality to replace it
+     */
     public function generateLinkDescription(array $data): string
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('3.0.0', 'Will be removed without equivalent functionality to replace it.');
+        });
         return '';
     }
 
+    /**
+     * @deprecated 3.0.0 Will be removed without equivalent functionality to replace it
+     */
     public function LinkTypeTile(): string
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('3.0.0', 'Will be removed without equivalent functionality to replace it.');
```

Comment:
```suggestion
     * @deprecated 3.0.0 Use getMenuTitle() instead.
     */
    public function LinkTypeTile(): string
    {
        Deprecation::withNoReplacement(function () {
            Deprecation::notice('3.0.0', 'Use getMenuTitle() instead.');
```
As with the others, I don't mind what the exact wording is so long as it matches the others.

Reply:
Updated